course/layouts/partials/video.html

```diff
@@ -7,4 +7,21 @@
     {{ .Params.about_this_resource_text | safeHTML }}
   </div>
   {{ partial "youtube_player.html" (dict "youtubeKey" $youtubeKey "captionsLocation" $captionsLocation "transcriptLocation" $transcriptLocation) }}
+  {{if $captionsLocation}}
```

Comment:
Shouldn't this be `{{ if $transcriptLocation }}`?

Reply:
No,` transcriptLocation` is the location of the downloadable pdf transcript.

`captionsLocation` is the location of the  .vtt captions file which is used by videojs and this plugin to display captions

Reply:
A comment would have been fine, but that works too